docs/api-docs/apps/tutorials/apps-id.md

```diff
@@ -5,10 +5,66 @@
 ### On This Page
 - [Find in Control Panel](#find-in-control-panel)

```

Comment:
[Headings and Titles](https://intranet.bigcommerce.com/display/SST/Headings+and+Titles)